src/test-run/index.js

```diff
@@ -261,3 +300,42 @@ ServiceMessages[CLIENT_MESSAGES.ready] = function (msg) {
 
     return new Promise((resolve, reject) => this.pendingRequest = { resolve, reject, responseTimeout });
 };
+
+ServiceMessages[CLIENT_MESSAGES.executePendingBrowserManipulation] = async function (msg) {
+    this.debugLog.driverMessage(msg);
```

Comment:
FYI, this question relates to https://github.com/DevExpress/testcafe/pull/552#discussion_r64570866